pyproject.toml

```diff
@@ -67,7 +67,7 @@ testing = [
     "pytest-cov",
     "pytest-timeout",
     "flexmock",
-    "deepdiff",
+    "deepdiff <= 7.0.1", # version 8.0.0 depends on numpy
```

Comment:
Could we be consistent and use either `< 8.0.0` (probably better, in case of a patch release in the future) or `<= 7.0.1` everywhere?